lib/scaffold.dart

```diff
@@ -618,7 +624,7 @@ class BackdropScaffoldState extends State<BackdropScaffold>
     );
   }
 
-  Future<bool> _willPopCallback(BuildContext context) async {
+  Future<bool?> _willPopCallback(BuildContext context) async {
     if (isBackLayerRevealed) {
       concealBackLayer();
       return null;
```

Comment:
Could `_willPopCallback` also return something non-null?

Reply:
Resolved in https://github.com/fluttercommunity/backdrop/pull/86/commits/c77c561f2f754576242980531f8e08710632e2a0